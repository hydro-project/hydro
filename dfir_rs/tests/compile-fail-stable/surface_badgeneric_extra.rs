../compile-fail/surface_badgeneric_extra.rs