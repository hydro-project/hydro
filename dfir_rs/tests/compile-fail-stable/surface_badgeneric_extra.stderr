error: Unknown lifetime generic argument `'a`, expected `'none`, `'loop`, `'tick`, `'static`, or `'mutable`.
 --> tests/compile-fail-stable/surface_badgeneric_extra.rs:6:42
  |
6 |         source_iter(0..10) -> identity::<'a, usize>() -> for_each(std::mem::drop);
  |                                          ^^
